### PR TITLE
Changes required to expose a user's Management Portal permissions via an API call

### PR DESCRIPTION
**Pull request by cobusc**
`/ops/user_management_portal_permissions/{user_id}`

### CODE REVIEW

File: management_layer/integration.py [hunk 1]
```diff
@@ -671,6 +698,43 @@ class Implementation(AbstractStubClass):
             user_site_role_labels_aggregated = await request.app["operational_api"].get_user_site_role_labels_aggregated(user_id, site_id)
             return user_site_role_labels_aggregated.to_dict()
 
+    # get_users_with_roles_for_domain -- Synchronisation point for meld
```
Comment (DavidOberholzer) [inline management_layer/integration.py]: These were my additions last week, what are they doing here :confused:?

File: management_layer/integration.py [hunk 2]
```diff
@@ -651,6 +622,62 @@ class Implementation(AbstractStubClass):
                                                   site=site_id, domain=domain_id, nocache=nocache)
         return {"has_permissions": result}
 
+    # get_user_management_portal_permissions -- Synchronisation point for meld
+    @staticmethod
+    @require_permissions(all, [("urn:ge:access_control:permission", "read"),
+                               ("urn:ge:access_control:resource", "read"),
+                               ("urn:ge:access_control:roleresourcepermission", "read"),
+                               ], target_user_field=1)
```
[the remark this answers is not in the document]
Comment (cobusc) [reply management_layer/integration.py]: The `target_user_field` is used to identify the user _for whose data_ the call is being made.
If the user making the request (as identified by the token in the request)  is the same user as the one for which data is being requested, then permission is implied.

File: management_layer/integration.py (hunk 1 above)
Comment (cobusc) [reply management_layer/integration.py]: The functions were not in the same place as the generated code (stubs.py), so I just moved it.

File: management_layer/integration.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (DavidOberholzer) [reply management_layer/integration.py]: Ah yes now I remember, thats perfect then :+1:

**Pull request merged**